perceval/backends/core/gerrit.py

```diff
@@ -520,6 +526,7 @@ def setup_cmd_parser(cls):
         group.add_argument('--ssh-port', dest='port',
                            default=PORT, type=int,
                            help="Set SSH port of the Gerrit server")
+        group.add_argument('--ssh-path', dest='path', help="Set SSH path")
```

Comment:
The name of this argument and variables around the code should be different. The SSH parameter uses the SSH identity file, so I think it will be better something like `ssh-id-filepath`. What do you think?

Reply:
OK, will address tomorrow.

Reply:
Addressed, tests added, not squashed yet, I'll squash when CI tests pass.